MedtrumKit/Packets/SetTimeZonePacket.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  SetTimeZonePacket.swift
+//  MedtrumKit
+//
+//  Created by Bastiaan Verhaar on 27/02/2025.
+//
+
+struct SetTimeZonePacketResponse {}
+
+class SetTimeZonePacket : MedtrumBasePacket {
+    typealias T = SetTimeZonePacketResponse
+    
+    let commandType: UInt8 = CommandType.SET_TIME_ZONE
+    
+    func getRequestBytes() -> Data {
```

Comment:
Ensure you do the workaround here for this issue:
https://github.com/nightscout/AndroidAPS/pull/2831

Reply:
Done!